tests/core/test_read_manifest.py

```diff
@@ -23,25 +23,14 @@ def test_read_manifest_fields():
     assert chip2.get('input', 'rtl', 'verilog', field='copy') is False
 
 
-def test_read_sup():
-    '''Test compressed read/write'''
-
-    chip = siliconcompiler.Chip('foo')
-    chip.input('foo.v')
-    chip.write_manifest('tmp.sup.gz')
-
-    chip2 = siliconcompiler.Chip('foo')
-    chip2.read_manifest('tmp.sup.gz')
-    assert chip2.get('input', 'rtl', 'verilog', step='import', index=0) == ['foo.v']
-
-
 # Use nostrict mark to prevent changing default value of [option, strict]
 @pytest.mark.nostrict
 def test_modified_schema(datadir):
     '''Make sure schema has not been modified without updating defaults.json'''
 
     # gets default from schema
     chip = siliconcompiler.Chip('test')
+    chip.schema._remove('dependency', 'siliconcompiler')
```

Comment:
This seems to be a bit the idea of this test but I know it fails with the dependency....

Reply:
I'll need to double check this one

Reply:
rewrote the test